runtime/doc/windows.txt

```diff
@@ -940,6 +940,11 @@ CTRL-W CTRL-Z					*CTRL-W_CTRL-Z* *:pc* *:pclose*
 		window editing it).  The command fails if any "Preview" buffer
 		cannot be closed.  See also |:close|.
 
+							*:fc* *:fclose*
+:[count]fc[lose][!]
+		Close [count]th floating window by zindex order. '!' to close
+		all floating windows.
+
```

Comment:
I don't think this blocking is in the correct section. This section is about the preview window.

Reply:
we don't have a popup chapter :thinking:

Reply:
okay